repos/linker.proj

```diff
@@ -20,7 +20,7 @@
     <BuildPackagesCommandArgs>$(BuildPackagesCommandArgs) $(ProjectDirectory)src/ILLink.Tasks/ILLink.Tasks.csproj</BuildPackagesCommandArgs>
     <BuildPackagesCommand>$(DotNetCliToolDir)dotnet $(BuildPackagesCommandArgs)</BuildPackagesCommand>
 
-    <PackagesOutput>$(ProjectDirectory)artifacts/packages/illink_$(Configuration)/Shipping/</PackagesOutput>
+    <PackagesOutput>$(ProjectDirectory)artifacts/packages/illink_$(Configuration)/NonShipping/</PackagesOutput>
```

Comment:
Let both through with this bit I just noticed in other projects?

```xml
  <Target Name="SetOutputList" AfterTargets="Package" BeforeTargets="GatherBuiltPackages">
    <ItemGroup>
      <PackagesOutputList Include="$(ShippingPackagesOutput)" />
      <PackagesOutputList Include="$(NonShippingPackagesOutput)" />
    </ItemGroup>
  </Target>
```

Reply:
I'm not sure adding both is worthwhile, because later in the project, we're installing the version we built, which needs to use `PackagesOutput` to locate the package.